RSDKv3/Userdata.cpp

```diff
@@ -287,8 +287,9 @@ void InitUserdata()
 #endif
 
 #if RETRO_PLATFORM == RETRO_OSX
-    sprintf(gamePath, "%s/RSDKv3", getResourcesPath());
-    sprintf(modsPath, "%s/RSDKv3/", getResourcesPath());
+    getResourcesPath(gamePath, sizeof(gamePath));
+    //sprintf(gamePath, "%s", getResourcesPath());
```

Comment:
You can remove this comment

Reply:
Forgot I left that in until I committed so sure